_data/simple-icons.json

```
@@ -3563,6 +3563,12 @@
             "hex": "4A86CF",
             "source": "https://flathub.org/"
         },
+        {
+            "title": "Flatpak",
+            "hex": "4A90D9",
+            "source": "https://flatpak.org/",
```

Since you've picked the icon from the ZIP archive, the source should be the press page which links to it.

```suggestion
            "source": "https://flatpak.org/press/",
```

---

I swear I did link it correctly but then again I was half-asleep during creating this PR so ehhh... I probably overlooked this one. ;p

---

Haha, no worries. :smile: